src/index.html

```diff
@@ -1,13 +1,29 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
+    <title>Antisnake</title>
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
+  </head>
+  <body>
+    <div class="container">
+      <div class="container__block--1 block">1</div>
```

Comment:
```suggestion
      <div class="block"></div>
```
why do you need this class, if you add styles through `n-th-child`?
you can also add numbers in blocks through scss loop